plugins/secret-squirrel/src/tasks/secret-squirrel.ts

```diff
@@ -0,0 +1,17 @@
+import { Task } from '@dotcom-tool-kit/types'
+import { fork } from 'child_process'
+import { hookFork, waitOnExit } from '@dotcom-tool-kit/logger'
+
+const secretSquirrelPath = require.resolve('.bin/secret-squirrel')
+
+export default class SecretSquirrel extends Task {
+  static description = 'run Secret Squirrel to check for secrets in your repo'
```

Comment:
**praise**: not just an empty string in the `description` field!
**thought**: filling these in should probably be part of the documentation work?